config/webpacker.yml

```diff
@@ -41,7 +41,8 @@ development:
 
 test:
   <<: *default
-  compile: true
+  compile: false
```

Comment:
So, we never let webpack to compile on the fly for test and we rely on `assets:precompile`. 
And for dev, we create docu explaining that we need webpack-dev-server (`rake webpack:dev`). Would you add the docu here or separate PR?